### PR TITLE
Spark3.3 support

### PR DESCRIPTION
**Pull request by luxu1-ms**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ There are three version sets of the connector available through Maven, a 2.4.x,
 | Spark 2.4.x compatible connnector | `com.microsoft.azure:spark-mssql-connector:1.0.2` | 2.11 |
 | Spark 3.0.x compatible connnector | `com.microsoft.azure:spark-mssql-connector_2.12:1.1.0` | 2.12 |
 | Spark 3.1.x compatible connnector | `com.microsoft.azure:spark-mssql-connector_2.12:1.2.0` | 2.12 |
+| Spark 3.3.x compatible connnector | `com.microsoft.azure:spark-mssql-connector_2.12:1.3.0` | 2.12 |
```
Comment (luxu1-ms) [inline README.md:19]: will update to 2.13

**Review by shivsood — APPROVED** [review]

**Comment by luxu1-ms** [comment]
@microsoft-github-policy-service agree [company="microsoft"]

**Comment by luxu1-ms** [comment]
@microsoft-github-policy-service agree
@microsoft-github-policy-service agree company="Microsoft"

**Comment by dalmat36** [comment]
@luxu1-ms hello, curious when we can expect a new release with spark 3.3 support? Thanks!

**Comment by shivsood** [comment]
@dalmat36 targeting a beta by Jan. This branch should work fine with spark 3.3. Build with 'mvn clean package', and use the jar. Let us know any issue on this thread. Thanks!

File: README.md (hunk 1 above)
Comment (shivsood) [reply README.md]: why did we change this. Scala version for Spark 3.3 is till 2.12

**Comment by ganeshchand** [comment]
Any updates on when this PR will be merged?

**Pull request closed without merging**